lib/pymedphys/_experimental/streamlit/apps/iviewdb.py

```diff
@@ -19,7 +19,8 @@
 from pymedphys._imports import streamlit as st
 from pymedphys._imports import xmltodict
 
-from pymedphys._streamlit.utilities import dbf, misc
+from pymedphys._experimental.streamlit.utilities import dbf
+from pymedphys._streamlit.utilities import misc
```

Comment:
Why is this not also in experimental?

Reply:
This is a module that has its roots within the MetersetMap GUI originally:

https://github.com/pymedphys/pymedphys/blob/165dca77fa61d763f25f468dd6f8226a5f4603f9/lib/pymedphys/_streamlit/apps/metersetmap/_mosaiq.py#L20

https://github.com/pymedphys/pymedphys/blob/165dca77fa61d763f25f468dd6f8226a5f4603f9/lib/pymedphys/_streamlit/apps/metersetmap/_mosaiq.py#L37-L39

It is part of the non-experimental applications. It has some degree of coverage via e2e testing.

Reply:
Here is it utilised within the Monaco section of MetersetMap:

https://github.com/pymedphys/pymedphys/blob/165dca77fa61d763f25f468dd6f8226a5f4603f9/lib/pymedphys/_streamlit/utilities/monaco.py#L9-L20

And the DICOM section:

https://github.com/pymedphys/pymedphys/blob/165dca77fa61d763f25f468dd6f8226a5f4603f9/lib/pymedphys/_streamlit/apps/metersetmap/_dicom.py#L90-L94